models/demos/t3000/mixtral8x7b/tt/model_config.py

```diff
@@ -21,7 +21,7 @@ class TtModelArgs:
     norm_eps = 1e-05
     vocab_size = 32000
 
-    max_batch_size = 32  # default
+    max_batch_size = 8  # default
```

Comment:
why not 32?

Reply:
max_batch_size here is defined based on the max length of 32k. This ensures that if the user tries to use the model with a new script it won't go above the limitations.

But I'll add an update with a comment on the supported batch sizes <-> seqlens to the model config, so end users are aware. 👍

Reply:
thank you, it is clearer now.